source/ranges.tex

```diff
@@ -12882,3 +12919,648 @@
+  requires @\libconcept{same_as}@<Allocator, void> || @\libconcept{convertible_to}@<const Alloc&, Allocator>
+  void* operator new(size_t size, allocator_arg_t, const Alloc& alloc, const Args&...);
+
+template<class This, class Alloc, class... Args>
+  requires @\libconcept{same_as}@<Allocator, void> || @\libconcept{convertible_to}@<const Alloc&, Allocator>
+  void* operator new(size_t size, const This&, allocator_arg_t, const Alloc& alloc,
+                       const Args&...);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+Let \tcode{A} be
+\begin{itemize}
+\item
+\tcode{Allocator}, if it is not \tcode{void},
+\item
+\tcode{Alloc} for the overloads with a template parameter \tcode{Alloc}, or
+\item
+\tcode{allocator<void>} otherwise.
+\end{itemize}
+Let \tcode{B} be \tcode{allocator_traits<A>::template rebind_alloc<U>}
+where \tcode{U} is an unspecified type whose size and alignment
+are both \tcode{__STDCPP_DEFAULT_NEW_ALIGNMENT__}.
+
+\pnum
+\mandates
+\tcode{allocator_traits<B>::pointer} is a pointer type.
+
+\pnum
+\effects
+Initializes an allocator \tcode{b} of type \tcode{B} with \tcode{A(alloc)},
+for the overloads with a function parameter \tcode{alloc},
+or with \tcode{A()} otherwise.
+Uses \tcode{b} to allocate storage for the smallest array
+of \tcode{U} sufficient to provide storage for
+a coroutine state of size \tcode{size}, and
+unspecified additional state necessary to ensure that
+\tcode{operator delete} can later deallocate this memory block
+with an allocator equal to \tcode{b}.
+
+\pnum
+\returns
+A pointer to the allocated storage.
+\end{itemdescr}
+
+\indexlibrarymember{operator delete}{generator::promise_type}%
+\begin{itemdecl}
+void operator delete(void* pointer, size_t size) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\expects
+\tcode{pointer} was returned from an invocation of
+one of the above overloads of \tcode{operator new}
+with a \tcode{size} argument equal to \tcode{size}.
+
+\pnum
+\effects
+Deallocates the storage pointed to by \tcode{pointer}
+using an allocator equivalent to that used to allocate it.
+\end{itemdescr}
+
+\rSec2[coro.generator.iterator]{Class template \tcode{generator::\exposid{iterator}}}
+
+\begin{codeblock}
+namespace std {
+  template<class Ref, class V, class Allocator>
+  class generator<Ref, V, Allocator>::@\exposid{iterator}@ {
+  public:
+    using value_type = @\exposid{value}@;
+    using difference_type = ptrdiff_t;
+
+    @\exposid{iterator}@(@\exposid{iterator}@&& other) noexcept;
+    @\exposid{iterator}@& operator=(@\exposid{iterator}@&& other) noexcept;
+
+    @\exposid{reference}@ operator*() const noexcept(is_nothrow_copy_constructible_v<@\exposid{reference}@>);
+    @\exposid{iterator}@& operator++();
+    void operator++(int);
+
+    friend bool operator==(@\exposid{iterator}@ i, default_sentinel_t);
+
+  private:
+    coroutine_handle<promise_type> @\exposid{coroutine_}@; // \expos
+  };
+}
+\end{codeblock}
+
+\indexlibraryctor{generator::iterator}%
```

Comment:
Replace all `{generator::iterator}%` with `{generator::\exposid{iterator}}%`

Reply:
Not in index entries.

Reply:
Indeed. No iterator/sentinel in `<ranges>` does this. But I got confused since some motion's PRs are doing this.